server/subscriptions.go

```diff
@@ -16,6 +16,11 @@ const (
 	SUBSCRIPTIONS_KEY = "subscriptions"
 )
 
+type SubWithRepo struct {
```

Comment:
0/5 on this one, but `SubWithRepo` seems like a redundant struct just to be able to access the repository for the subscription in question. I wonder if we could solve this more holistically by storing the repository in the existing `Subscription` struct (in addition to the repository being part of the data structure)? I could see cases where I want access to all the data.

Reply:
also one thing that came to my mind: Is it so that currently it is not possible to subscribe to same repository in two different channels (because its map with key = repo)? In my opinion that is not perfect solution

Reply:
@lieut-data so you think that we could leave `Subscriptions` as is but add repo information also to `Subscription` struct? Then repo information is basically in two places, in map key and in subscription struct. However, at least I do not see that as big problem?